.github/workflows/postcommit.yml

```
@@ -0,0 +1,96 @@
+  postcommit:
+    name: Postcommit
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.11", "3.12"]
+    steps:
+      - uses: actions/checkout@v6
+      - name: Delete cancelled runs in the same concurrency group
+        env:
+          GH_TOKEN: ${{ github.token }}
+        run: |
```

actionlint (1.7.10)

"github.head_ref" is potentially untrusted. avoid using it directly in inline scripts. instead, pass it through an environment variable. see https://docs.github.com/en/actions/reference/security/secure-use#good-practices-for-mitigating-script-injection-attacks for more details

(expression)